.gitlab-ci.yml

```diff
@@ -79,6 +79,18 @@ make-coq-latest:
     - tags
     - merge_requests
 
+.dune-build:
+  stage: build
+  image: coqorg/coq:dev
+  script:
+    - opam switch "$COMPILER_EDGE"
+    - opam pin -y add dune https://github.com/ejgallego/dune.git#coq+compose_local
```

Comment:
Do we really need an unreleased version of Dune?

Reply:
It should work fine with recent versions, but the PR needs updating.

Reply:
Updated.